moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1925,9 +1925,6 @@ double moveit::core::RobotState::computeCartesianPath(const JointModelGroup* gro
   traj.clear();
```

Comment:
See code right above this fold:

You said "This removes the minimum addition of 4 extra waypoints for every waypoint." but the code that determines the number of steps 

```
   unsigned int steps = (test_joint_space_jump ? 5 : 1) + (unsigned int)floor(distance / max_step);
```

remains unchanged